spec/support/spec_config.rb

```diff
@@ -296,7 +296,11 @@ def client_cert_path
   end
 
   def local_client_pem_path
-    "#{ssl_certs_dir}/client.pem"
+    if (algo = ENV['OCSP_ALGORITHM'])&.empty?
+      "#{ssl_certs_dir}/client.pem"
+    else
+      Pathname.new("#{spec_root}/support/ocsp/#{algo}/server.pem")
```

Comment:
Would be great if this had a comment explaining the logic behind each of the branches.